tests/integration/aws_s3/test_aws_s3.py

```diff
@@ -60,20 +87,31 @@ def setup_method(self, method):
         object_file = f"file_{random.randint(0, 1000)}.txt"
         csv_file = f"file_{random.randint(0,1000)}.csv"
 
-        self._bucket_path = "s3://" + self._bucket_name
-        self._object_path = object_dir + "/" + object_file
-        self._df_path = object_dir + "/" + csv_file
-
-        self._object_url = self._bucket_path + "/" + self._object_path
-        self._df_url = self._bucket_path + "/" + self._df_path
-        self._blob_url = self._object_url + ".blob"
-
-        logger.info(f"Object URL: {self._object_url}")
-
-    def _perform_aws_s3_tests(self, secrets=None):
-        data_item = mlrun.run.get_dataitem(self._object_url, secrets=secrets)
+        self.s3 = {}
+        self.s3["s3"] = self._make_target_names(
+            "s3://", self._bucket_name, object_dir, object_file, csv_file
+        )
+        self.s3["ds"] = self._make_target_names(
+            "ds://s3ds_profile/", self._bucket_name, object_dir, object_file, csv_file
+        )
+
+        logger.info(f'DS Object URL: {self.s3["ds"]["object_url"]}')
+        logger.info(f'S3 Object URL: {self.s3["s3"]["object_url"]}')
+
+        project = mlrun.get_or_create_project("default", "./")
```

Comment:
Shouldn't this be
```suggestion
        project = mlrun.get_or_create_project(self.project_name))
```
?

Reply:
Or
```suggestion
        project = mlrun.get_or_create_project(self.project_name, "./")
```

Reply:
will remove